hasher-matcher-actioner/webapp/src/pages/settings/ThreatExchangeSettingsTab.tsx

```diff
@@ -25,13 +25,30 @@ import {
   deleteDataset,
 } from '../../Api';
 
+type privacyGroupType = {
```

Comment:
Dunno whether we are following a specific convention or not, but Types should be UpperCamelCase, IMO.

https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html seems to follow the same.

Reply:
nit: And the 'Type' suffix is unnecessary. 

Because this is distinct from the actual PrivacyGroup domain objects, I understand you'd want to clearly make this separate. In previous projects, I have used ThreatExchangeSettingsPrivacyGroupState style type/class names. 

It is verbose, but explicit. YMMV.